python/lsst/meas/astrom/astrometry.py

```diff
@@ -279,9 +288,14 @@ def solve(self, exposure, sourceCat):
 
         self.log.info(
             "Matched and fit WCS in %d iterations; "
-            "found %d matches with scatter = %0.3f +- %0.3f arcsec" %
+            "found %d matches with mean and scatter = %0.3f +- %0.3f arcsec" %
             (iterNum, len(tryRes.matches), tryMatchDist.distMean.asArcseconds(),
-                tryMatchDist.distStdDev.asArcseconds()))
+             tryMatchDist.distStdDev.asArcseconds()))
+        if tryMatchDist.distMean.asArcseconds() > self.config.maxMeanOffsetArcsec:
+            self.log.fatal("Assigning as a fit failure: mean offset on sky = %0.3f arcsec > %0.3f "
+                           "config.maxMeanOffsetArcsec" % (tryMatchDist.distMean.asArcseconds(),
```

Comment:
"%" -> ","  - Use the lazy log formatter, instead of formatting the string directly.